src/components/templates/TalkShowStudio/TalkShowStudio.tsx

```diff
@@ -94,56 +94,56 @@ export const TalkShowStudio: FC<TalkShowStudioProps> = ({ venue }) => {
       ),
     [userOnStageSharingScreen?.data, remoteUsers, venue.id]
   );
-  const remoteCameraTrack = useMemo(
-    () =>
-      remoteUsers.find(
-        ({ uid }) =>
-          `${uid}` ===
-          userOnStageSharingScreen?.data?.[`${venue.id}`]?.cameraClientUid
-      ),
-    [userOnStageSharingScreen?.data, remoteUsers, venue.id]
-  );
 
   const remoteUsersPlayers = useMemo(() => {
+    const uniqueUsersIds = new Set();
     const setRemoteUserAvatar = (remoteUserId: number | string) => {
       if (!venue.id) return;
-      const remoteUser = stage.peopleOnStage.find(
+
+      return stage.peopleOnStage.find(
         ({ data }) =>
           data?.[`${venue.id}`]?.cameraClientUid === `${remoteUserId}`
       );
-      return remoteUser;
     };
```

Comment:
What is the purpose of this function? It doesn't seem to 'set' anything? It finds/returns something?